File: Copilot/1. Algoritmo Básico Ordenamiento por Burbuja/C++/burbuja.cpp
#include <iostream>
#include <vector>
#include <sstream>
using namespace std;

void bubbleSort(vector<int>& arr) {
    int n = arr.size();
    for (int i = 0; i < n-1; i++)
        for (int j = 0; j < n-i-1; j++)
            (arr[j] > arr[j+1])
                swap(arr[j], arr[j+1]);
}

int main() {
    cout << "Ingresa números separados por espacio: ";
    string line;
    getline(cin, line);
    stringstream ss(line);
    int num;
    vector<int> arr;

    while (ss >> num) arr.push_back(num);

    bubbleSort(arr);

    cout << "Lista ordenada: ";
    for (int n : arr) cout << n << " ";
    cout << endl;
    return 0;
}

File: ChatGPT/1.Algoritmo Básico Ordenamiento por Burbuja/python/burbuja.py
def bubble_sort(arr):
    n = len(arr)
    for i in range(n):
        for j in range(0, n-i-1):
            if arr[j] > arr[j+1]:
                arr[j], arr[j+1] = arr[j+1], arr[j]
    return arr

if __name__ == "__main__":
    entrada = input("Ingrese números separados por espacios: ")
    numeros = list(map(int, entrada.strip().split()))
    print("Lista ordenada:", bubble_sort(numeros))

